src/modules/dogs/views/dogs.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <b-container>
+    <b-card-group columns>
+      <b-skeleton-wrapper :loading="!dogimageData || !dogimageData.length">
+        <template #loading>
+          <b-card v-for="i in 20" :key="i">
+            <b-skeleton-img></b-skeleton-img>
+            <b-card-body>
+              <b-row>
+                <b-skeleton width="85%"></b-skeleton>
+              </b-row>
+            </b-card-body>
+          </b-card>
+        </template>
+        <b-card
+          :key="i"
+          img-top
+          img-height="250"
+          v-for="(dogs, i) in dogimageData"
+          :title="`Dog image`"
+          :img-src="dogs"
+          :img-alt="`dog_image-${i}`"
+        >
+        </b-card>
+      </b-skeleton-wrapper>
+    </b-card-group>
+  </b-container>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      dogimageData: [],
+    };
+  },
+  async mounted() {
+    await this.$store.dispatch("dogs/getDogsData", true);
+
+    this.dogimageData = this.$store.state["dogs"].dogsData;
```

Comment:
Why are you not using getters??

Reply:
if some point in time i want to change dogsData variable name to dogData i will have to change it everywhere..i am trying to avoid code repetition scenarios.

Reply:
getter and state can be mapped to another variable so this safety measure is pointless. 
https://vuex.vuejs.org/api/#mapstate
https://vuex.vuejs.org/api/#mapgetters

Reply:
ok will use getters

Reply:
plz check